lib/index.js

```diff
@@ -29,7 +27,7 @@ exports.plugin = {
 };
 
 
-internals.vision = class Vision {
+const Vision = internals.vision = class Vision {
```

Comment:
I don't think this jives with the style guide—perhaps just reference `internals.Vision` as needed.